scripts/deps-downloader.fsx

```diff
@@ -0,0 +1,18 @@
+let baseUrl = "https://github.com/MonoGame/MonoGame.Dependencies/raw/master/SDL"
+let pathToProjectBin = sprintf "../MarsBaseBuilder/bin/%s" fsi.CommandLineArgs.[2]
```

Comment:
That's not a good idea. Consider user running the script from the project root, or from some other directory.

Replace it with `"__SOURCE_DIRECTORY__/../MarsBaseBuilder/bin/%s"`.